fabmukunzi/certificate-manager---Fabrice
# [chore]: setup liquibase

#### PR description

- I refactored my database configurations to use liquibase instead of hibernate for flexible database management

#### Checklist:


#### How To Test Changes

- Checkout branch `ch-setup-liquibase`
- Open terminal and type `cd backend` , `mvn clean`,  and then `quarkus dev`
- Check if the tables are created
